openid-4-verifiable-presentations-1_0.md

```diff
@@ -1900,88 +1900,137 @@ these constraints. The Wallet is not controlled by the Verifier and the Verifier
+
+The Verifier SHOULD ensure that information collection purpose is sufficiently specific and communicated before collection. For example, the purpose is shown to the End-User before the presentation request is sent to the Wallet.
+
+If the Wallet has indications that the Verifier is requesting data that it is not entitled to, the Wallet SHOULD warn the End-User and potentially stop processing.
+
+## Selective Disclosure {#selective-disclosure}
+
+Selective disclosure is a data minimization technique that allows for sharing only the specific information needed from
+a Credential without revealing all of the claims contained in that Credential.
+
+The DCQL helps facilitate selective disclosure by allowing the Verifier to specify the claims it is interested in,
+allowing the Wallet to disclose only the claims that are relevant to the Verifier's request.
+
+Some Credential formats support selective disclosure and a salted-hash based approach is one common approach.
+
+### DCQL Value Matching {#dcql-value-matching}
 
 When using DCQL `values` to match the expected values of claims, the fact that a
-claim within a certain credential matched a value or did not match a value might
+claim within a certain Credential matched a value or did not match a value might
 already leak information about the claim value. Therefore, Wallets MUST take
 precautions against leaking information about the claim value when processing
 `values`. This SHOULD include, in particular:
 
-- ensuring that a Verifier, in the response, cannot distinguish between the case where a user did
-  not consent to releasing the credential and the case where the claim value did
+- ensuring that a Verifier, in the response, cannot distinguish between the case where an End-User did
+  not consent to releasing the Credential and the case where the claim value did
   not match the expected value, and
 - preventing repeated or "silent" requests leaking data to the Verifier without
   the user's consent by ensuring that all requests, even if no response can be
-  sent by the Wallet due to a `values` mismatch, require some form of user
+  sent by the Wallet due to a `values` mismatch, require some form of End-User
   interaction before a response is sent.
 
 In both cases listed here, it needs to be considered that returning an error
 response can also leak information about the processing outcome of `values`.
 
-# Privacy Considerations
+### Strictly Necessary Claims {#strictly_necessary}
 
-Many privacy considerations are specific to the credential format and associated proof type used in any particular presentation.
-This section focuses on privacy considerations that are specific to the presentation protocol with some treatment also
-given to considerations that apply across some common credential formats.
+Verifiers SHOULD design DCQL queries that request only the minimal set of claims and Credentials needed to fulfill the specified purposes.
```

Comment:
```suggestion
Verifiers SHOULD use DCQL queries that request only the minimal set of claims and Credentials needed to fulfill the specified purposes.
```

Minor but many verifiers may used pre-developed queries that are set by ecosystems, hence aren't the designer.